### ISSUE TITLE
Video constraints cannot be applied to external camera

### ISSUE THREAD

**Issue by mekya** [ant-media/Ant-Media-Server#3273]
### Short description
Video constraints cannot be applied to external camera even if external camera supports the specified resolutions. 

### Steps to reproduce
1. Specify video constraints for resolution such as 480x360(other than default) in index.html
2. Connect an external usb camera to the computer
3. Open the index.html

### Expected behavior
Camera resolution should be 480x360 or the specified resolution. 

### Actual behavior
Camera resolution is the default.

**Comment by eneskuluk** [comment]
I have did some testing with external webcam, here is my findings:
Resolution that i applied worked flawlessly.
Check the following screenshots:
Publisher page:
![image](https://user-images.githubusercontent.com/54481799/119023555-bf4cea80-b9aa-11eb-88f5-f3ab79815ddb.png)

Player page:

![image](https://user-images.githubusercontent.com/54481799/119023635-dc81b900-b9aa-11eb-9ff7-a42767ab0ce1.png)

Even if i change the active webcam/scene, nothing has changed and resolution was still intact to what i set: 480x360.

**Comment by dominik-schmid** [comment]
Hi @eneskuluk 

I had the problem that @mekya described.

I've tested your solution with my internal webcam and it works without problems. Unfortunately, I don't have access to the external camera which caused this problem. I will test it again as soon as I got the equipment back.

But your post was still already helpful because I learned where to set the constraints. Until now I was setting them manually or [with the official WebRTC samples](https://webrtc.github.io/samples/src/content/getusermedia/resolution/). It worked with the internal camera but not with the external one. Maybe the way I was setting the constraints and your default code were interfering with each other.

I will get back to you asap but it will take a few days or weeks.
